### ISSUE TITLE
Only last UITextView subscriber receives events

### ISSUE THREAD

**Issue by corysullivan** [CombineCommunity/CombineCocoa#48]
First of all, thanks for all of your hard work. I appreciate the additions to Combine in both the Cocoa and Ext packages. They have helped bridge the gap between RxSwift and Combine.
**Describe the bug**

**To Reproduce**
I have an issue with UITextView. If I have multiple subscriptions to the same textView `textPublisher`, only the last subscription receives events. I have tested other controls, and they work as expected, so I believe this issue is specific to UITextView. I have a sample project that I could pass along; however, the issue is easy to reproduce.
1. Create a UItextView in the storyboard
2. Add an outlet to viewcontroller
3. Create two subscriptions to the `textPublisher`
4. run the app and add input to the UItextView
5. Only the last subscription receives updates

**Expected behavior**
I expect both subscriptions to receive text input updates

**Additional context**
 - CombineCocoa version (0.2.2)

Example:
```
class ViewController: UIViewController {
    private var disposables = Set<AnyCancellable>()
    @IBOutlet var textView: UITextView!

    override func viewDidLoad() {
        super.viewDidLoad()

        textView.textPublisher
            .sink { text in
                print("Subscription-1 \(text)")
            }
            .store(in: &disposables)

        textView.textPublisher
            .sink { text in
                print("Subscription-2 \(text)")
            }
            .store(in: &disposables)
    }
}
```
produces the following output:

```
Subscription-1 Optional("")
Subscription-2 Optional("")
Subscription-2 Optional("T")
Subscription-2 Optional("Th")
Subscription-2 Optional("Thi")
Subscription-2 Optional("This")
Subscription-2 Optional("This ")
Subscription-2 Optional("This i")
Subscription-2 Optional("This is")
Subscription-2 Optional("This is ")
Subscription-2 Optional("This is a")
Subscription-2 Optional("This is a ")
Subscription-2 Optional("This is a m")
Subscription-2 Optional("This is a me")
Subscription-2 Optional("This is a mes")
Subscription-2 Optional("This is a mess")
Subscription-2 Optional("This is a messa")
Subscription-2 Optional("This is a messag")
Subscription-2 Optional("This is a message")
```

**Comment by freak4pc** [comment]
Hey there ! 
Can you check the version in the main branch ? 

I believe this was addressed in #47

**Comment by corysullivan** [comment]
@freak4pc yes it's fixed on the main branch. Thanks!
